tests/unit_tests/cuda/mask_store_cuda_kernel.hpp

```diff
@@ -39,6 +39,17 @@ using ring = ring2<>;
 using single = single3<0>;
 using trapezoid = trapezoid2<>;
 
+/** Enumerate different mask types for convenience
+ **/
+enum mask_ids : unsigned int {
```

Comment:
I guess this lines are leftover?

Reply:
No, it is needed, since all mask ID ```enums``` are now local to a specific detector

Reply:
Oh right. it's called in mask_store_cuda.cpp